src/java.base/share/classes/java/lang/Module.java

```diff
@@ -195,7 +195,7 @@ public String getName() {
      *
      * @return The class loader for this module
      */
-    public ClassLoader getClassLoader() {
+    public @Nullable ClassLoader getClassLoader() {
```

Comment:
It would be nice if this were better documented. It does turn out that this can be null, I think for the bootstrap/system/whatever classloader:

```
$ cat ClassLoaderForModule.java 
public class ClassLoaderForModule {
  public static void main(String[] args) {
    System.out.println(Object.class.getModule().getClassLoader());
  }
}

$ javac ClassLoaderForModule.java && java ClassLoaderForModule
null
```